cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help

```diff
@@ -70,8 +70,8 @@ EXAMPLES
        Generate both the Ballerina client and service file.
```

Comment:
```suggestion
       Generate both the Ballerina client and service file.
```
We have added the full stop in some places only. Please check all the places and either keep them or remove them to maintain consistency.